backend/python/app/services/implementations/user_service_mg.py

```diff
@@ -0,0 +1,345 @@
+import firebase_admin.auth
+
+from ..interfaces.user_service import IUserService
+from ...models.user_mg import User
+from ...resources.user_dto import UserDTO
+
+
+class UserService(IUserService):
+    """
+    UserService implementation with user management methods
+    """
+
+    def __init__(self, logger):
+        """
+        Create an instance of UserService
```

Comment:
Decided to use the Sphinx format for docstrings, seems to be a popular and choice and makes it easy to autogenerate docs: https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

cc @alexguo8 could you please update the docstrings in EmailService?